extension/chrome/elements/compose-modules/compose-draft-module.ts

```diff
@@ -104,10 +104,12 @@ export class ComposeDraftModule extends ViewModule<ComposeView> {
         msgData.pwd = undefined; // not needed for drafts
         const sendable = await new EncryptedMsgMailFormatter(this.view, true).sendableMsg(msgData, pubkeys);
         this.view.S.cached('send_btn_note').text('Saving');
-        if (this.view.threadId) { // reply draft
-          sendable.body['text/plain'] = `[cryptup:link:draft_reply:${this.view.threadId}]\n\n${sendable.body['text/plain'] || ''}`;
-        } else if (this.view.draftId) { // new message compose draft with known draftid
-          sendable.body['text/plain'] = `[cryptup:link:draft_compose:${this.view.draftId}]\n\n${sendable.body['text/plain'] || ''}`;
+        if (!(sendable.body instanceof Uint8Array)) {
```

Comment:
For me - clean up with `this.view.recipientsModule.isSmimeRecipientPresent()`

Reply:
Actually this must never happen. I think. Because we are only using our own public key for drafts, which (for now) is always opgp:
```ts
const pubkeys = [{ isMine: true, email: msgData.from, pubkey: primaryKi.public }];
```